tests/gnrc_dhcpv6_client_stateless/README.md

```diff
@@ -0,0 +1,30 @@
+# `gnrc_dhcpv6_client_stateless` test
```

Comment:
Would it make sense to add some documentation on the functionality into the test and add the graph for the Automaton?

![image](https://user-images.githubusercontent.com/675644/135821468-13cf2be2-43c1-45f2-997a-1e2d58deca54.png)

(created by running the test with the following patch)
```
diff --git a/tests/gnrc_dhcpv6_client_stateless/tests-as-root/01-run.py b/tests/gnrc_dhcpv6_client_stateless/tests-as-root/01-run.py
index e9e532418d..0363ec3ffd 100755
--- a/tests/gnrc_dhcpv6_client_stateless/tests-as-root/01-run.py
+++ b/tests/gnrc_dhcpv6_client_stateless/tests-as-root/01-run.py
@@ -166,6 +166,7 @@ def build_reply_headers(pkt):
 
 def testfunc(child):
     iface = os.environ["TAP"]
+    StatelessDHCPv6Test.graph()
     StatelessDHCPv6Test(child, iface=iface).run()
 
 
```

Reply:
The visualization is now included in the README as well as a description of what is actually tested. I also added some documentation comments to the python file. Let me know if anything should be changed :)